Use proper username

TheEvilSkeleton opened the pull request:
Hey, thanks a lot for adding me as a contributor! I changed my username since I use `TheEvilSkeleton` as my actual username and `Proprietary Chrome-chan` for the memes.

qarmin commented on czkawka_gui/src/gui_structs/gui_about.rs:
```diff
@@ -51,7 +51,7 @@ impl GuiAbout {
             "Michael Grigoryan",
             "Nikita Karamov",
-            "Proprietary Chrome-chan",
+            "TheEvilSkeleton",
```
Can you add in comment `// Proprietary Chrome-chan`? 
This will help me to not forget to replace the name with the proper one in future changes

TheEvilSkeleton commented:
I added a comment with Proprietary Chrome-chan. I also added a `.mailmap` file to make the output of `git shortlog` better. FYI, `git shortlog` outputs your name a couple of times:

```
TheEvilSkeleton@TheMainDesktop ~/P/czkawka (update-name) [16]> git shortlog -sne --author="Rafał Mikrut"
   197  Rafał Mikrut <41945903+qarmin@users.noreply.github.com>
   140  Rafał Mikrut <mikrutrafal54@gmail.com>
    77  Rafał Mikrut <mikrutrafal@protonmail.com>
```

If you want, I can add your name in the mailmap file with the preferred email address.
